cmd/helm/load_plugins.go

```diff
@@ -154,7 +154,7 @@ func callPluginExecutable(pluginName string, main string, argv []string, out io.
 func manuallyProcessArgs(args []string) ([]string, []string) {
 	known := []string{}
 	unknown := []string{}
-	kvargs := []string{"--kube-context", "--namespace", "-n", "--kubeconfig", "--kube-apiserver", "--kube-token", "--kube-as-user", "--kube-as-group", "--kube-ca-file", "--registry-config", "--repository-cache", "--repository-config", "--insecure-skip-tls-verify", "--tls-server-name"}
+	kvargs := []string{"--kube-context", "--namespace", "-n", "--kubeconfig", "--kube-apiserver", "--kube-token", "--kube-as-user", "--kube-as-group", "--kube-ca-file", "--registry-config", "--repository-cache", "--repository-config", "--kube-insecure-skip-tls-verify", "--kube-tls-server-name"}
```

Comment:
It appears there is both a [`--kube-insecure-skip-tls-verify`](https://github.com/helm/helm/blob/098eed2c07c548e15fa383e11e507486ab3cb410/pkg/cli/environment.go#L150) and a [`--insecure-skip-tls-verify`](https://github.com/helm/helm/blob/098eed2c07c548e15fa383e11e507486ab3cb410/cmd/helm/push.go#L97) flag.

Can both be listed in this slice?

Reply:
@mattfarina I think not. `--insecure-skip-tls-verify` There is no corresponding environment variable